keps/draft-20180426-security-conformance.md

```diff
@@ -0,0 +1,465 @@
+---
+kep-number: draft-20180426
+title: Security Conformance
+authors:
+  - "@easeway"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-testing
+  - sig-api-machinery
+  - sig-network
+  - sig-node
+  - sig-cluster-lifecycle
+reviewers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+  - "@davidopp"
+approvers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+editor: "@easeway"
+creation-date: 2018-04-26
+status: provisional
+---
+
+# Security Conformance
```

Comment:
I like this proposal, but it seems seems like there are some things missing, for example which admission controllers should be enabled. Basically more of the stuff discuss in
https://docs.google.com/document/d/1jAcsC4sLgEV9__TdgJrMvPa3G73G62tFtMcKQgeIlHM/edit
(though I think you did hit most of the things mentioned in that doc).

BTW I would suggest adding a section discussing the tradeoff between having the test just directly check whether a particular policy is installed, vs. check that the behavior dictated by the policy is actulally enforced. It seems you chose the latter; I think that's fine but it would be good to justify that choice.

Reply:
Yes. I will add a section in alternatives.